File: TOKI/Course_Pemrograman Dasar/05 Percabangan/G.py
array = [ int(x) for x in input().split() ]

jarak = 0
if array[0] > array[2]:
    jarak += array[0] - array[2]
else:
    jarak += array[2] - array[0]

if array[1] > array[3]:
    jarak += array[1] - array[3]
else:
    jarak += array[3] - array[1]

print(jarak)

File: TOKI/Course_Pemrograman Dasar/05 Percabangan/G.cpp
#include <cstdio>
#include <cstdlib>

struct titik {
    int x, y;
};

int main() {
    titik titik1;
    titik titik2;
    scanf("%d %d", &titik1.x, &titik1.y);
    scanf("%d %d", &titik2.x, &titik2.y);

    int jarak = abs(titik1.x-titik2.x) + abs(titik1.y-titik2.y);
    printf("%d\n", jarak);

    return 0;
}